docs/docs/reference/dstack.yml/task.md

```diff
@@ -520,3 +518,9 @@ The `task` configuration type supports many other options. See below.
 
     * `volume-name:/container/path` for network volumes
     * `/instance/path:/container/path` for instance volumes
+
+## `retry`
+
+#SCHEMA# dstack._internal.core.models.profiles.ProfileRetry
+    overrides:
+      show_root_heading: false
```

Comment:
It's already documented for tasks, line 455.

If you decide to go with the old version from line 455, please also move the retry section to the top of the dev-environment and service references to keep the order of sections consistent.

Reply:
Thanks! I'll move it before `resources` since it's quite important.